osbs/api.py

```diff
@@ -521,14 +524,19 @@ def _check_labels(self, repo_info):
                       utils.Labels.LABEL_TYPE_VERSION]:
             try:
                 _, req_labels[label] = labels.get_name_and_value(label)
+
+                if label in strict_labels and not req_labels[label]:
+                    required_missing = True
+                    logger.error("required label doesn't have explicit value in Dockerfile : %s",
+                                 labels.get_name(label))
             except KeyError:
                 required_missing = True
                 logger.error("required label missing from Dockerfile : %s",
                              labels.get_name(label))
 
         try:
             _, release_value = labels.get_name_and_value(utils.Labels.LABEL_TYPE_RELEASE)
-            if not RELEASE_LABEL_FORMAT.match(release_value):
+            if release_value and not RELEASE_LABEL_FORMAT.match(release_value):
```

Comment:
Could this block also be moved to the `else` clause?

Reply:
I'd rather leave it here as it is more visible than in else after except